criu/config.c

```diff
@@ -691,6 +692,7 @@ int parse_options(int argc, char **argv, bool *usage_error,
 		{ "pre-dump-mode",		required_argument,	0, 1097},
 		{ "file-validation",		required_argument,	0, 1098	},
 		{ "lsm-mount-context",		required_argument,	0, 1099	},
+		{ "network-lock",		required_argument,	0, 1100	},
```

Comment:
We also need to add a description in `criu --help` and [`man criu`](https://github.com/checkpoint-restore/criu/blob/f29a02a395083941fea8d5f09ccf7efacdd60836/Documentation/criu.txt)

Reply:
I was thinking of adding that later when I add the ```--network-lock nftables``` option, but now that you pointed it out, I think I should add the ```nftables``` option + help/docs here because the rest depends on the option's existence anyway.
I will update it right away.